Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,169 +1,165 @@
 {
-  "object": {
```

Comment:
This change isn't needed I don't think? maybe rebase will dump it

Reply:
Weird one, should be fixed now.